utils.js

```diff
@@ -94,6 +97,49 @@ function getRedirectUri (query, param = 'redirect_uri') {
     : `https://${VEGA_URL}/`
 }
 
+const loadEnvVars = async () => {
+  const config = readFileSync(`./config/${process.env.ENVIRONMENT}.env`)
+  .toString()
+  .split('\n')
+  .map(line => line.split('='))
+
+  config.forEach(([key, value]) => { process.env[key] = value })
```

Comment:
Rolling your own ```require('dotenv').config({ path: `./config/${process.env.ENVIRONMENT}.env` })``` ? Some env vars contain `=` in their values (base64 encoded values end in them 2/3 of the time), so you may be dropping part of the value with above implementation.

Reply:
Replaced with dotenv